how-to/maintain/backup-and-restore/mydumper-lightning.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Use `Mydumper` and `TiDB Lightning` for Backup and Restoration
```

Comment:
Please replace all `mydumper` with Mydumper and replace all `TiDB Lightning` with TiDB Lightning.
```suggestion
title: Use Mydumper and TiDB Lightning for Backup and Restoration
```